### PR TITLE
Add STOPTIMEOUT to deb init script

### PR DESCRIPTION
**Pull request by repeatedly**
Need a way to control stop timeout. Related to https://github.com/treasure-data/omnibus-td-agent/pull/47

### CODE REVIEW

**Comment by repeatedly** [comment]
@yyuu Do you see any concern? If not, I will merge it.

File: templates/etc/init.d/deb/td-agent [hunk 1]
```diff
@@ -117,7 +120,7 @@ do_stop() {
   #   2 if daemon could not be stopped
   #   other if a failure occurred
   local RETVAL=0
-  start-stop-daemon --stop --quiet --retry=TERM/30/KILL/5 --pidfile "${TD_AGENT_PID_FILE}" --name ruby || RETVAL="$?"
+  start-stop-daemon --stop --quiet --retry=TERM/$STOPTIMEOUT/KILL/5 --pidfile "${TD_AGENT_PID_FILE}" --name ruby || RETVAL="$?"
```
Comment (yyuu) [inline templates/etc/init.d/deb/td-agent]: This is really trivial but argument should be quoted; `--retry="TERM/${STOPTIMEOUT}/KILL/5"`

**Comment by yyuu** [comment]
Looks good.

**Comment by repeatedly** [comment]
Applied. After test passed, I will merge.

**Comment by eagletmt** [comment]
:+1:

**Pull request merged**